src/main/resources/i18n/messages.properties

```diff
@@ -866,49 +863,71 @@ project.samples.filter.by-endDate=To
-project.export.organization.description=Name of organization submitting these samples.
-project.export.namespace.title=Identifier Namespace
-project.export.namespace.description=Prefix to use for submission identifiers in NCBI.
-project.export.release_date.title=Release Date
-project.export.release_date.description=Submission will not be made public until after the chosen release date.
-project.export.biosample.title=BioSample ID
-project.export.biosample.description=NCBI BioSample to add files to.  This sample must be created in NCBI prior to this submission.
-project.export.library_name.title=Library Name
-project.export.library_name.description=The submitter's name for this library. <a href="http://www.ncbi.nlm.nih.gov/books/NBK54984/table/SRA_Glossary_BK.T._library_descriptor_te/" target="_blank">NCBI Documentation</a>.
-project.export.library_strategy.title=Library Strategy
-project.export.library_strategy.description=Sequencing technique intended for this library. <a href="http://www.ncbi.nlm.nih.gov/books/NBK54984/table/SRA_Glossary_BK.T._library_descriptor_te/" target="_blank">NCBI Documentation</a>.
-project.export.library_selection.title=Library Selection
-project.export.library_selection.description=Whether any method was used to select for or against, enrich, or screen the material being sequenced. <a href="http://www.ncbi.nlm.nih.gov/books/NBK54984/table/SRA_Glossary_BK.T._library_descriptor_te/" target="_blank">NCBI Documentation</a>.
-project.export.library_source.title=Library Source
-project.export.library_source.description=The type of source material that is being sequenced. <a href="http://www.ncbi.nlm.nih.gov/books/NBK54984/table/SRA_Glossary_BK.T._library_descriptor_te/" target="_blank">NCBI Documentation</a>.
-project.export.instrument_model.title=Instrument Model
-project.export.instrument_model.descripton=The sequencing platform used to produce the data.  <a href="http://www.ncbi.nlm.nih.gov/books/NBK54984/table/SRA_Glossary_BK.T._platform_descriptor_t/" target="_blank">NCBI Documentation</a>.
-project.export.accession=NCBI Accession
-project.export.warning.select=One or more files must be selected for upload.
-project.export.submitter=Submitter
-project.export.status=Upload Status
-project.export.project=Project
-project.export.sidebar.title=SRA Submission
-project.export.table.title=Project Exports
-project.export.table.state=Upload State
-project.export.table.project=Project
-project.export.table.samples=Exported Samples
-project.export.warning.nofiles=Warning: Sample {0} has no files.  It cannot be submitted to NCBI.
-
 NcbiExportTable.loader-error=A project id is required.
 NcbiExportDetailsView.loader-error=No project id or export id.
 
+# TYPE: NcbiSubmissionRequest (webapp/resources/js/apis/export/ncbi.ts)
+NcbiSubmissionRequest.projectId=BioProject ID
+NcbiSubmissionRequest.projectId.description=BioProject to submit files to.  This project must be created in NCBI prior to this submission.
```

Comment:
Looks like there is an extra space between the two sentences in the description

Reply:
Updated in eb7900e4cc9fd2f11a8efe501597b42d66ca80c9